public/app/features/dashboard/components/ShareModal/SharePublicDashboard/ConfigPublicDashboard/SettingsBar.tsx

```diff
@@ -0,0 +1,45 @@
+import { css } from '@emotion/css';
+import React, { useCallback, useState } from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { useStyles2 } from '@grafana/ui';
+
+import { SettingsBarHeader } from './SettingsBarHeader';
+
+export interface Props {
```

Comment:
does it make sense to change it to 
```suggestion
import { SettingsBarHeader, Props as SettingsBarHeaderProps } from './SettingsBarHeader';

export interface Props extends Pick<SettingsBarHeaderProps, 'headerElement' | 'title'> {
  children: React.ReactNode;
}

```

so, there's correlation between the props?

Reply:
Sure, I like that idea 😊. I'd do it the other way around, since SettingsBarHeader is more specific than SettingsBar and is its child in this case

Reply:
Ah no, but those are being passed down exactly because the child needs them, so it makes sense that the parent's props refer those of the child. I see!